tests/test_api.py

```diff
@@ -53,7 +53,7 @@ def test_invalid_operation_does_stop_application_to_setup():
         FlaskApi(TEST_FOLDER / "fixtures/op_error_api/swagger.yaml",
                  base_path="/api/v1.0", arguments={'title': 'OK'})
 
-    with pytest.raises(ResolverError):
+    with pytest.raises(ValueError):
```

Comment:
Connexion internal exceptions are not exposed to users. They will see the actual errors, which is much better.